build/presubmit_unit.yaml

```diff
@@ -56,9 +56,3 @@ steps:
   args: ["build/gen_check.sh"]
   waitFor: ["-"]
   id: Generated files check
-
```

Comment:
Why is this removed?

Reply:
causing failure and we no longer care about policygen

Reply:
got it